[#2107] Checkboxes/radio buttons are now IE9 friendly

mjschranz opened the pull request:
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2107-ie9-radio-buttons-have-no-selected-state-in-editor

k88hudson commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/76b430f92bffe39/output.txt

k88hudson commented on css/editors.less:
```diff
@@ -328,6 +328,10 @@
   }
 
+  input[type="checkbox"]:checked {
+    box-shadow: 0 0 0 2px #3FB58E;
```
lessify please:
`@green`

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/76b430f92bffe39/output.txt

Total script time: 1.65 mins

Server is running on http://pr959.butyr.org/templates/basic/
Run the Test Suite: http://pr959.butyr.org/test/

jbuck commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a781e0a2b9cd143/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a781e0a2b9cd143/output.txt

Total script time: 0.92 mins

Server is running on http://pr959.butyr.org/templates/basic/
Run the Test Suite: http://pr959.butyr.org/test/

jbuck commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5d64588cfcdfcca/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5d64588cfcdfcca/output.txt

Total script time: 1.82 mins

Server is running on http://pr959.butyr.org/templates/basic/
Run the Test Suite: http://pr959.butyr.org/test/

mjschranz committed:
[#2107] Styles forced checkboxes previous to be too smaller in IE9
However, setting them to larger heights/widths doesn't seem to affect them in other browsers. Now they will have a visible checked state.
Needed to remove the border that shows in IE9, plus moved them closer to the text that's there for each one.

mjschranz committed:
Margin top needed to be applied to all radio/checkboxes and not only ones in the checkbox group
